quran/quran.com-frontend-next
# Add tafsirs route for a verse

### Summary
This PR adds a tafsir route to show tafsirs of a specific Ayah. The user can also select multiple tafsirs globally and will automatically show when he selects a tafsir for an Ayah.

### Test Plan
1. Click of Ayah's action menu, click on Tafsirs.
2. It should navigate to a page that shows the default Tafsirs of the verse selected.
3. Click on the Developer Utility and change the default tafsirs.
4. The newly selected tafsirs should show on the spot on the tafsirs page.

### Screenshots

| Before | After |
| ------ | ------ |
|<img width="185" alt="Screen Shot 2021-08-11 at 16 21 34" src="https://user-images.githubusercontent.com/15169499/129004388-338d33dc-0eb6-41a0-ac0a-e332600f0448.png">  | <img width="180" alt="Screen Shot 2021-08-11 at 16 23 01" src="https://user-images.githubusercontent.com/15169499/129004394-f4a70c6b-3105-4631-a902-cf4fbf26871d.png"> |
|  | <img width="1440" alt="Screen Shot 2021-08-11 at 16 19 55" src="https://user-images.githubusercontent.com/15169499/129004370-4d508fa3-b2c7-4d05-be83-5aaf088ac673.png"> |
|  | <img width="505" alt="Screen Shot 2021-08-11 at 16 23 26" src="https://user-images.githubusercontent.com/15169499/129004438-32a3fdc4-a009-4689-820f-5d502e8df231.png"> |
